lib/inbox.rb

```diff
@@ -237,32 +237,45 @@ def latest_cursor
       "message" => Inbox::ExpandedMessage,
     }
 
-    def _build_exclude_types(exclude_types)
-      exclude_string = "&exclude_types="
+    def _build_exclude_types(types)
+      exclude_string = "&exclude_types=" + _build_types_list(types)
+    end
+
+    def _build_include_types(types)
+      include_string = "&include_types=" + _build_types_list(types)
+    end
 
-      exclude_types.each do |value|
+    def _build_types_list(types)
+      type_string = ""
+      types.each do |value|
         count = 0
```

Comment:
Very good question - I had completely ignored it. It looks like it must be left over from some old code. I just removed it.